### DISCUSSION TITLE
Firefox/Chrome don't work correctly

### DISCUSSION THREAD

**Discussion by DonDaMck** [neutrinolabs/xrdp#2851] [Q&A]
Hello-

I recently imaged a Raspberry Pi 4 using the latest Raspberry Pi OS for handling xrdp sessions.  When I connect to it via RDP from a Windows 10 system, it works fine except for browsers.  See image:

<img width="623" alt="xrdp_issue" src="https://github.com/neutrinolabs/xrdp/assets/150482309/5559789b-34d9-4d71-ade9-8b237e1e23a5">

Any ideas about what's going on?  I'd appreciate assistance.  

Best,
Don

**Comment by matt335672** [comment]
Could be duplicate of #2430. Check the suggestions in the askubuntu link.

**Reply by Draugor** [reply]
i currently have the same issue (on my pi5) and checking in #2430 i run "chromium-browser --disable-gpu" which works 
for firefox i could only get it to run when i used "--save-mode" after figuring that out i disabled the hardware-acceleration in the firefox settings and it worked too, though i still get "[GFX1-]: vaapitest: ERROR
[GFX1-]: vaapitest: VA-API test failed: failed to initialise VAAPI connection." errors even with that

**Reply by matt335672** [reply]
Thanks @Draugor - that's useful to know.

**Comment by LaudixGit** [comment]
This was my solution
https://www.reddit.com/r/raspberry_pi/comments/166ht48/corrupted_remote_desktop_image/

`sudo nano /etc/X11/xrdp/xorg.conf`

Scroll down to this line, comment it out, and replace with truncated version
    #Option "DRMDevice" "/dev/dri/renderD128"
    Option "DRMDevice" ""
